site/content/en/latest/user/private-key-provider.md

```diff
@@ -0,0 +1,306 @@
+---
+title: "Using Envoy Private Key Provider for TLS termination"
+---
+
+To allow the accelerate the private key operation through hardware or protect the private key through the hardware, the [Envoy Private Key Provider](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#extensions-transport-sockets-tls-v3-privatekeyprovider) is added to Envoy.
```

Comment:
```suggestion
TLS operations can be accelerated or the private key can be protected using specilized hardware. This can be leveraged in Envoy using [Envoy Private Key Provider](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#extensions-transport-sockets-tls-v3-privatekeyprovider).
```